### PR TITLE
Addition of draft 'anagram' exercise for GNU APL

### PR DESCRIPTION
**Pull request by Russtopia**
EDIT: I figured out how to run `configlet lint` on this new exercise and it now passes cleanly for me, at least locally. PR updated.

I am still relatively new to APL, so hopefully this solution is not a horrible one. :smiley: I would love to see GNU APL exercises 'go live' on the Exercism platform!

### CODE REVIEW

**Comment by marnen** [comment]
Thanks for the contribution! This looks interesting and I’ll have a closer look later. You will have to get the checks to pass in order for this to be merged, though.

File: config.json [hunk 1]
```diff
@@ -120,10 +120,33 @@
...
+    "paradigm/functional",
+    "paradigm/array",
+    "execution_mode/interpreted",
+    "platform/windows",
+    "platform/mac",
+    "platform/linux",
+    "runtime/interpreter_or_lib",
+    "used_for/data_analysis",
+    "used_for/mathematics",
+    "used_for/finance"
```
Comment (Russtopia) [inline config.json:150]: I admit I just made up these tags; is there an official list that should be used?

File: docs/SNIPPET.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env apl --script
+
+∇z←hello
+  z←'Hello, World!'
+∇
+
```
Comment (Russtopia) [inline docs/SNIPPET.txt:6]: Just mimicking what the Go track appeared to put in this file

File: config.json (hunk 1 above)
Comment (Russtopia) [reply config.json]: Ran `configlet` and looked up what currently valid tags to use instead; updated.

**Comment by Russtopia** [comment]
> Thanks for the contribution! This looks interesting and I’ll have a closer look later. You will have to get the checks to pass in order for this to be merged, though.

Hi, I think I've fixed the issues identified by the `configlet lint` in this and the other two PRs I submitted; please advise if I am using the wrong version or have missed something though. Unsure how to move these PRs forward.

```$ configlet --version
4.0.0-alpha.25```

**Comment by marnen** [comment]
Thank you! I’m very busy this week but will look at this and your other pull requests as soon as I have a minute. I really appreciate you taking the time to contribute.

**Pull request closed without merging**